lib/rust/frp/src/io/keyboard.rs

```diff
@@ -36,6 +36,22 @@ impl Side {
 
 
 
+// =========================
+// === ArrowKeyDirection ===
+// =========================
+
+/// The four directions of the arrow keys on a typical keyboard.
+#[derive(Copy, Clone, Debug)]
+#[allow(missing_docs)]
+pub enum ArrowKeyDirection {
+    Up,
+    Down,
+    Left,
+    Right,
```

Comment:
It is already defined in this file. Why are you redefining it? I see the definition of `ArrowLeft`. If you want to have a struct only with these enums, it is not merged with that other definition - is it intentional to have 2 structs representing the same things (e.g. "an arrow down button")?